nixos/modules/services/security/bitwarden_rs/default.nix

```
@@ -36,7 +36,7 @@ in {
     };
 
     config = mkOption {
-      type = attrsOf (nullOr (either (either bool int) str));
+      type = attrsOf (nullOr (oneOf [ bool int str ]));
```

```suggestion
      type = attrsOf (oneOf [ (nullOr bool) int str ]);
```
or `oneOf [ null bool int str ]`

---

I don't think `null` is a type? And I prefer the `nullOr (oneOf ...)`, because it gives the meaning of "either unset or one of these"

---

I've suggested it so it would be compatible with my another suggestion - https://github.com/NixOS/nixpkgs/pull/65728#issuecomment-519816354

But I'm fine with this as well.